region/crateria/west.json

```diff
@@ -1516,8 +1737,52 @@
                         {"canComeInCharged": {
                           "fromNode": 1,
                           "framesRemaining": 0,
-                          "shinesparkFrames": 90
-                        }}
+                          "shinesparkFrames": 90,
+                          "excessShinesparkFrames": 17
+                         }}
+                      ],
+                      "additionalObstacles": ["A"],
+                      "note": "Enter through the top of the door to reach all the wall to the shot block wall."
+                    }
+                  ]
+                },
+                {
+                  "name": "Shinespark in Room",
+                  "notable": false,
+                  "requires": [],
+                  "obstacles": [
+                    {
+                      "id": "B",
```

Comment:
there may be other strats that could be added. Blue suit through B for free, which saves a PB for A. fixme is fine